integration_tests/commands/type_test.go

```diff
@@ -0,0 +1,57 @@
+package commands
+
+import (
+	"testing"
+
+	dstore "github.com/dicedb/dice/internal/store"
+	"gotest.tools/v3/assert"
+)
+
+func TestType(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
+	testCases := []struct {
+		name     string
+		commands []string
+		expected []interface{}
+		setup    func(store *dstore.Store)
+	}{
+		{
+			name:     "TYPE for non-existent key",
+			commands: []string{"TYPE k1"},
+			expected: []interface{}{"none"},
+		},
+		{
+			name:     "TYPE for key with String value",
+			commands: []string{"SET k1 v1", "TYPE k1"},
+			expected: []interface{}{"OK", "string"},
+		},
+		{
+			name:     "TYPE for key with List value",
+			commands: []string{"LPUSH k1 v1", "TYPE k1"},
+			expected: []interface{}{"OK", "list"},
+		},
+		{
+			name:     "TYPE for key with Set value",
+			commands: []string{"SADD k1 v1", "TYPE k1"},
+			expected: []interface{}{int64(1), "set"},
+		},
+		{
+			name:     "TYPE for key with Hash value",
+			commands: []string{"HSET k1 field1 v1", "TYPE k1"},
+			expected: []interface{}{int64(1), "hash"},
+		},
```

Comment:
Could you please add a testcase for data created using `SETBIT` Command as well  test case for number of arguments. This will complete tests for this command.

Reply:
Added the test cases for SETBIT , BITOP and number of arguments.